### DISCUSSION TITLE
Can tui-editor icons be replaced with different icons?

### DISCUSSION THREAD

**Discussion by verr0t** [taiga-family/taiga-ui#4520] [Q&A] [answered]
Hi, is there any possibility to replace the icons of tui-editor component?

**Comment by waterplea** [comment]
Not in an easy way. You would need to do a mapper of names to different icons as shown here:
https://taiga-ui.dev/icon-set#material

**Reply by verr0t** [reply]
And does that means the editor's toolbox is not customizable? Thank you btw.

**Reply by waterplea** [reply]
Not sure what you mean by that.

**Reply by verr0t** [reply]
Like to have access in the toolbox, and change icons directly from there or to add custom parts.

**Reply by waterplea** [reply]
You can add custom parts:
https://taiga-ui.dev/editor/custom-tool/paste-emoji

**Comment by waterplea** [comment, marked as answer]
Different icons can now be provided through DI:
https://github.com/taiga-family/editor/blob/main/projects/editor/common/editor-options.ts#L21
